src/automations/checkInactivity.js

```diff
@@ -102,64 +107,72 @@ async function scrapeInactiveIssues(client, references, issues) {
   const iterator = issues[Symbol.iterator]();
 
   for (let issue of iterator) {
-    const inactiveLabel = issue.labels.find(label => {
-      return label.name === client.cfg.activity.inactive;
-    });
-    if (inactiveLabel) continue;
-
-    let time = Date.parse(issue.updated_at);
     const issueNumber = issue.number;
     const repoName = issue.repository.name;
     const repoOwner = issue.repository.owner.login;
     const issueTag = `${repoName}/${issueNumber}`;
 
-    if (time < references.get(issueTag)) time = references.get(issueTag);
+    if (references.get(issueTag).review) {
+      const inactiveLabel = issue.labels.find(label => {
+        return label.name === client.cfg.activity.inactive;
+      });
+      if (inactiveLabel) continue;
 
-    const active = client.cfg.activity.check.repositories.includes(issueTag);
+      let time = Date.parse(issue.updated_at);
+      let prUpdateTime = references.get(issueTag).time;
```

Comment:
Should be a `const`; we don't change it. Also `prUpdateTime` isn't really camelcase?

Reply:
I changed `prUpdateTime` to `pullReqUpdateTime`. Also, `time` would not be a const because in the following if statement the value of `time` can be potentially updated, which is why `npm run test` returns an error.

Reply:
I changed `prUpdateTime` to `pullReqUpdateTime`, and made it a `const`. I'll be submitting a PR now :)